### PR TITLE
Add option for toggling update checks

### PR DESCRIPTION
**Pull request by voczi**
See: https://forums.ankiweb.net/t/how-to-disable-update-check-on-windows/47401/12

### CODE REVIEW

**Comment by voczi** [comment]
on a second thought, maybe it would be more appropriate to put this setting in the network tab

File: ftl/core/preferences.ftl [hunk 1]
```diff
@@ -41,6 +41,7 @@ preferences-theme-follow-system = Follow System
 preferences-theme-light = Light
 preferences-theme-dark = Dark
 preferences-v3-scheduler = V3 scheduler
+preferences-enable-update-check = Enable checking for program updates
```
Comment (dae) [inline ftl/core/preferences.ftl:44]:
```suggestion
preferences-enable-update-check = Check for program updates
```

File: qt/aqt/main.py [hunk 2]
```diff
@@ -1431,7 +1431,8 @@ def show_menubar(self) -> None:
     def setup_auto_update(self, _log: list[DownloadLogEntry]) -> None:
         from aqt.update import check_for_update
 
-        check_for_update()
+        if aqt.mw.pm.enable_update_check():
```
Comment (dae) [inline qt/aqt/main.py:1434]: `update_check_enabled` or `check_for_updates` for the function / key name would be a bit clearer/more consistent

**Review by dae — COMMENTED** [review]
'Appearance' is probably a little better than the 'Syncing' tab - either under general or distractions, perhaps?

**Comment by voczi** [comment]
Should be all good now.

**Comment by dae** [comment]
Thank you!

**Pull request merged**